src/Symfony/Component/EventDispatcher/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.0
+---
+
+* Remove `LegacyEventDispatcherProxy`
```

Comment:
Shall we use a short sentence like this in code removal? Or duplicate the message in `UPGRADE-6.0.md` that also contains the `Use the ... instead`?

Reply:
In this particular case, I don't think that anything beyond the note about the removal is helpful. That class is just plain obsolete now.

Reply:
When we have one, mentioning the replacement in the removal' UPGRADE instruction is good yes